src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,93 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.*;
+
+class BasicValidationServiceTest {
+
+    ValidationService validationService;
+
+    @BeforeEach
+    void setUp() {
+        validationService = new BasicValidationService();
+    }
+
+    @Test
+    void shouldAcceptTheAmount() {
```

Comment:
```suggestion
    void shouldSuccessfullyValidateTheAmount() {
```
it's job is to validate, not accept

Reply:
Done.